server/routes/api/v1/surrenderRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from "express"
+import SurrenderApplication from "../../../models/SurrenderApplication.js"
+
+const surrenderRouter = new express.Router()
+
+surrenderRouter.post("/", async (req, res) => {
+  try {
+    const surrenderApplication = new SurrenderApplication(req.body)
+    await surrenderApplication.save();
+    res.status(201).json({ surrenderApplication: surrenderApplication })
```

Comment:
currently `req.body` consists of the data we need to both create a pet and add an application.  We need to access and use the `petData` to create a new pet and try to save it.  Then we can use the `surrenderApplicationData` along with the newley persisted pet's ID to create a new surrender application and save that

Reply:
Fixed.